docs/marionette.collectionview.md

```diff
@@ -457,6 +465,295 @@ myView.destroyChildren({checkEmpty: false}); // will not show emptyView
+var ListItemView = Marionette.View.extend({
+  tagName: 'li',
+  template: '#list-item-text'
+});
+
+var ListView = Marionette.View.extend({
+  tagName: 'ul',
+  className: 'list-unstyled',
+
+  childView: ListItemView
+});
+
+var list = new Backbone.Collection([
+  {id: 1, text: 'My text'},
+  {id: 2, text: 'Another Item'}
+]);
+
+var listView = new ListView({
+  collection: list
+});
+
+listview.render();
+```
+
+With the template:
+
+```
+<%- text %>
+```
+
+This will render the following:
+
+```html
+<ul class="list-unstyled">
+  <li>My text</li>
+  <li>Another Item</li>
+</ul>
+```
+
+### Rendering Tables
+
+Marionette 3 introduced a major improvement to `View` to make it possible to
+implement tables using only `View` and `CollectionView`. This section will
+demonstrate how to build a table in Marionette 3, with the equivalent in
+Marionette 2 using `CompositeView`.
+
+#### Tables Using Marionette 2
+
+**_The following code is deprecated and for demonstration purposes only _**
+
+To build a table in Marionette 2 requires the `CompositeView` which we'll build
+as such:
+
+```javascript
+var RowView = Marionette.LayoutView.extend({
+  tagName: 'tr',
+  template: '#table-row'
+});
+
+var TableView = Marionette.CompositeView.extend({
+  tagName: 'table',
+  className: 'table table-hover',
+  template: '#table',
+  childView: RowView,
+  childViewContainer: 'tbody'
+});
+
+var list = new Backbone.Collection([
+  {id: 1, text: 'My text'},
+  {id: 2, text: 'Another Item'}
+]);
+
+var myTable = new TableView({
+  collection: list
+});
+
+myTable.render();
+```
+
+Given the following `#table` and `#table-row` templates:
+
+```html
+<thead>
+  <tr>
+  <th>ID</th>
+  <th>Body</th>
+  </tr>
+</thead>
+<tbody></tbody>
+```
+
+```html
+<td><%- id %></td>
+<td><%- text %></td>
+```
+
+Will render the following:
+
+```html
+<table class="table table-hover">
+<thead>
+  <tr>
+  <th>ID</th>
+  <th>Body</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>1</td>
+    <td>My text</td>
+  </tr>
+  <tr>
+    <td>2</td>
+    <td>Another Item</td>
+  </tr>
+</tbody>
+</table>
+```
+
+A major downside of this method was that it was impossible to add extra regions
+inside the `CompositeView` - if a header item needed to be re-rendered based on
+user input, then the entire table must be re-rendered, or the DOM must be
+manipulated with `ui` items.
+
+To resolve this issue, Marionette 3 improves the `View` to make it possible to
+build tables without `CompositeView`.
+
+#### Tables Using Marionette 3
+
+Marionette 3 doesn't use `CompositeView` any more. We now build tables using
+`View`s and `regions`. The following code will render the same table as in
+[Marionette 2][#tables-using-marionette-2-compositeview]:
+
+```javascript
+var RowView = Marionette.View.extend({
+  tagName: 'tr',
+  template: '#row-template'
+});
+
+var TableBody = Marionette.CollectionView.extend({
+  tagName: 'tbody',
+  childView: RowView
+});
+
+var TableView = Marionette.View.extend({
```

Comment:
I'll have a look at the `View` documentation and clearly label it there.

I think we're going to need to keep the existing documentation with links to the updated versions of everything at least for the initial versions so ease the transition.